app/Community/Commands/SyncUserUlids.php

```diff
@@ -18,28 +17,22 @@ class SyncUserUlids extends Command
 
     public function handle(): void
     {
-        $total = User::whereNull('ulid')->count();
-
-        if ($total === 0) {
-            $this->info('No records need ULIDs.');
-
-            return;
-        }
+        $total = User::count();
```

Comment:
I realize this is meant to be a one-time command, but without the "whereNull" check, it must only be run once. The random millisecond salt will cause (statistically) every user to get a new ULID.

Reply:
Re-added these checks in latest.